readme: update

illwieckz opened the pull request:
Since Unvanquished's readme mention building the engine is only needed to run the game and link to this repository shipping readme, add some basic instructions about how to run a game or a server.

DolceTriade reviewed (commented):
Maybe link to dist/server.cfg for config options and our wiki page https://wiki.unvanquished.net/wiki/Server/Running

illwieckz commented on README.md:
````diff
@@ -105,3 +105,27 @@ For a 32-bit build use the `cross-toolchain-mingw32.cmake` toolchain file instea
+./daemon
+```
+
+If you want to run a server, you may want to use the non-graphical `daemonded` server binary and start a map this way:
````
Done.

illwieckz commented:
> @DolceTriade
> Maybe link to dist/server.cfg for config options and our wiki page https://wiki.unvanquished.net/wiki/Server/Running

I added a link to the Unvanquished readme explaining that (no need to duplicate the Unvanquished readme).

Changes since the last review:

- I added more links to Unvanquished.
- Some minor wording changes.
- I tried to add some ℹ️ and 💡️ emojis to mark some “ℹ️  Notable info” and some “💡️ Clever trick”. What do you think about it?
